startup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -o nounset
+set -o errexit
+
+destfile="/srv/config.txt"
+echo "$UPS_HOST" >> "$destfile"
```

Comment:
```suggestion
echo "$UPS_HOST" > "$destfile"
```

otherwise it will be appended everytime the server starts